src/findingconfig/until_test.go

```diff
@@ -5,6 +5,7 @@ import (
 	"time"
 
 	"github.com/cultureamp/ecrscanresults/findingconfig"
+	_ "github.com/hexops/autogold/v2"
```

Comment:
That's pretty whack, I had a quick browse of the repo and I didn't immediately find an issue relating to this. Is there an issue you found where this was the suggestion?

Reply:
No, I had to work it out

Reply:
The underlying issue seemed to stem from this change: https://github.com/hexops/autogold/pull/45, but I'm not 100% sure.

The Go test tool doesn't like it when a flag is present but not consumed by code, and writes a _long_ message to that effect. However, since `go test` generates a `main` function to run tests, and doesn't load `_test` files from other packages, if the `autogold` package isn't referenced by the package itself, the flag isn't consumed.

So when you run `go test ./... -update` and it iterates over all the packages, some of them write a failure about the unused flag, because they're not using "golden" files.

Reply:
Sorry for the short reply earlier.